jopa-impl/src/main/java/cz/cvut/kbss/jopa/model/metamodel/MetamodelBuilder.java

```diff
@@ -38,6 +42,7 @@ public class MetamodelBuilder {
     private final Map<Class<?>, AbstractIdentifiableType<?>> typeMap = new HashMap<>();
     private final Set<Class<?>> inferredClasses = new HashSet<>();
 
+    private final MultiValuedMap<IdentifiableType<?>, AnnotatedAccessor> annotatedAccessors = new HashSetValuedHashMap<>();
```

Comment:
If I am not mistaken, this is the only use of the Apache Commons collections library. In that case (and given that it is quite easy to replace it with a `Map<IdentifiableType<?>, Set<AnnotatedAccessor>>`, I would prefer removing the Commons collections dependency.

Reply:
SOLVED - dependency removed, replaced as suggested.